clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerProtocol.java

```diff
@@ -90,7 +91,8 @@ public static ByteBuffer serializeSubscription(PartitionAssignor.Subscription su
         return buffer;
     }
 
-    public static PartitionAssignor.Subscription deserializeSubscription(ByteBuffer buffer) {
+    public static PartitionAssignor.Subscription deserializeSubscription(ByteBuffer buffer,
```

Comment:
A little annoying to have to pass through `groupInstanceId` here, but I can't think of a better option. The only thing that comes to mind is passing through the full `JoinGroupResponseData.JoinGroupResponseMember` object, but that's annoying also.

Reply:
Yea, I guess it's a bit cleaner only passing in the fields we need.